README.md

```diff
@@ -34,6 +34,7 @@ The [`my-app/.templates`](.templates) dir contains some templates you can use in
 
 - `<APP_NAME>`: the application name
 - `<APP_SHORT_DESCRIPTION>`: a quick app description
+- `<APP_CATEGORY>`: the app category (used in the cozy-bar apps navigation)
```

Comment:
Well, if someone (not at cozy) wants to develop an app, he/she will be puzzled by this field. So, I think it's better to have at least one example